site/content/blog/2024-09-18-detachable-tabs/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "ZAP now has Detachable Tabs!"
```

Comment:
Ah, but it doesnt 😁 
ZAP as far as 99% of our users are concerned is the latest released version, which does not support detachable tabs.
If we're going to publish this blog before 2.16 then we have to make it very clear that its only dev/nightly/weekly that support this